composer.json

```diff
@@ -19,6 +19,8 @@
         "gisostallenberg/response-content-negotiation-bundle": "^0.5.1",
         "haydenpierce/class-finder": "^0.4.0",
         "league/html-to-markdown": "^4.8",
+        "php-http/guzzle6-adapter": "^1.0 || ^2.0",
```

Comment:
Could you also adapt the suggest message to the new situation, see
https://github.com/ConnectHolland/user-bundle/pull/68/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R64